<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(185, 28, 28);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>361</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    border-radius: 10px;
    padding: 8px 2px;
    font-size: 16px;
    font-weight: bold;
    border: none;
	color: rgb(0,0,128);
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollContent">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>345</width>
       <height>325</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="scrollLayout">
      <item>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:16px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px; color:#b91c1c;&quot;&gt;How to Prepare for Blood Donation &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px; color:#b91c1c;&quot;&gt;(If You’re Ready and Eligible)&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px;&quot;&gt;&lt;br /&gt;&lt;br /&gt;Great decision! Donating blood is a generous and life-saving act. To make the process smooth and safe, it’s best to start preparing 1–3 days before your donation appointment. Here’s what to do:&lt;br /&gt;&lt;br /&gt;⸻&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px; color:#b91c1c;&quot;&gt;1. 2–3 Days Before Your Donation:&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px;&quot;&gt;&lt;br /&gt;• Stay hydrated. Drink plenty of water each day. Good hydration makes your veins easier to find and reduces dizziness after donation.&lt;br /&gt;• Eat iron-rich foods. Boost your iron levels with red meat, leafy greens (like spinach), beans, nuts, eggs, and fortified cereals.&lt;br /&gt;• Avoid alcohol and fatty foods. Fatty meals can affect your blood quality, and alcohol may make you feel lightheaded.&lt;br /&gt;• If donating platelets: Stop taking aspirin at least 2 days before your appointment.&lt;br /&gt;&lt;br /&gt;⸻&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px; color:#b91c1c;&quot;&gt;2. 1 Day Before Your Donation:&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px;&quot;&gt;&lt;br /&gt;• Keep drinking water. Aim for an extra glass or two throughout the day.&lt;br /&gt;• Get a good night’s sleep. Feeling rested helps your body handle the donation and recover quickly.&lt;br /&gt;&lt;br /&gt;⸻&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px; color:#b91c1c;&quot;&gt;3. On the Day of Donation:&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px;&quot;&gt;&lt;br /&gt;• Eat a healthy meal 1–2 hours before donating. Never donate on an empty stomach.&lt;br /&gt;• Dress comfortably. Wear a shirt with sleeves that can be rolled up easily.&lt;br /&gt;• Bring your ID. Most donation centers will ask for a photo ID or donor card.&lt;br /&gt;• Let the staff know if it’s your first time. They’ll guide you through everything and make sure you’re comfortable.&lt;br /&gt;&lt;br /&gt;⸻&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px; color:#b91c1c;&quot;&gt;After You Donate:&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px;&quot;&gt;&lt;br /&gt;• Rest for 10–15 minutes at the donation site and have the snack and drink they provide.&lt;br /&gt;• Drink extra fluids for the rest of the day to stay hydrated.&lt;br /&gt;• Avoid heavy exercise or lifting for at least 24 hours.&lt;br /&gt;• If you feel lightheaded later, sit or lie down with your legs elevated and sip water.&lt;br /&gt;&lt;br /&gt;⸻&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px; color:#b91c1c;&quot;&gt;When Can You Donate Again?&lt;/span&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:16px;&quot;&gt;&lt;br /&gt;• Whole blood: Every 56 days&lt;br /&gt;• Platelets: Every 15 days (up to 24 times per year)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Общий стиль для всего окна */
QWidget {
    background-color: #b30000; /* насыщенный красный */
    font-family: 'Segoe UI', sans-serif;
    color: white;
}

/* Заголовки */
QLabel#titleLabel {
    font-size: 24px;
    font-weight: bold;
    color: white;
}

/* Подзаголовки и текст */
QLabel {
    font-size: 14px;
    color: white;
}

/* Поля ввода */
QLineEdit {
    background-color: white;
    border: 2px solid #ff4d4d;
    border-radius: 10px;
    padding: 6px;
    color: #333;
    font-size: 14px;
}

/* Фокус на поле */
QLineEdit:focus {
    border: 2px solid #ff1a1a;
}

/* Кнопки */
QPushButton {
    background-color: #b30000;
    color: white;
    font-weight: bold;
    font-size: 14px;
    border-radius: 12px;
    padding: 8px 16px;
    border: 2px solid white;
}

QPushButton:hover {
    background-color: #ffcccc;
    color: #800000;
}

QPushButton:pressed {
    background-color: #ffe6e6;
    border: 2px solid #ff9999;
}

/* Ошибки или уведомления */
QLabel#errorLabel {
    color: #ffcccc;
    font-size: 12px;
}

/* Кнопки внизу: Sign In / Sign Up переключение */
QPushButton#switchButton {
    background: transparent;
    color: white;
    font-size: 12px;
    text-decoration: underline;
    border: none;
}

QPushButton#switchButton:hover {
    color: #ff9999;
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="emergencyTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>343</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Preparation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
